the-telephone-app-fe/src/components/Draw/Draw.js

```diff
@@ -107,14 +107,24 @@ const Draw = ({ width = "815rem", height = "350rem" }) => {
 
   const [isClicked, setIsClicked] = useState(false);
 
+  // ===== to change line width
```

Comment:
Should capitalize first letter
Also 'to' is a bit redundant, could be just Change line width